x-pack/plugins/observability_solution/apm/public/components/app/service_overview/index.tsx

```diff
@@ -48,14 +48,16 @@ export function ServiceOverview() {
 
   const { start, end } = useTimeRange({ rangeFrom, rangeTo });
 
-  const hasLogsSignal =
-    serviceEntitySummary?.dataStreamTypes && isLogsSignal(serviceEntitySummary.dataStreamTypes);
+  const hasSignal =
```

Comment:
Aren't this the changes from [this other PR](https://github.com/elastic/kibana/pull/192263)?

Reply:
Yes, I merged it into mine because I needed it.... the other one will soon be merged and removed from this PR.